src/renderer/gl_engine/meson.build

```diff
@@ -51,9 +53,10 @@ else
     gl_target_profile = '-DTHORVG_GL_TARGET_GL=1'
 endif
 
+add_languages('c', required: true)
```

Comment:
![image](https://github.com/user-attachments/assets/1debc523-6465-455f-840c-8e0b769ea03a)

due to this, android has been failed, we don't need to use c language here?